numpy/core/tests/test_arrayprint.py

```diff
@@ -388,6 +389,7 @@ def test_wide_element(self):
             "[ 'xxxxx']"
         )
 
+    @dec.skipif(not HAS_REFCOUNT, "python has no sys.getrefcount")
```

Comment:
Maybe we should factor this out into a `@numpy.testing._needs_refcount` decorator? The need to import multiple objects + duplication of the message feels like a small missed abstraction to me.

Reply:
Ok, added a separate decorator for this + search & replace